test/ci_workflow_gen.go

```diff
@@ -56,6 +56,8 @@ var (
 		"vreplication_cellalias",
 		"vstream_failover",
 		"vreplication_v2",
+		"vstream_stoponreshard_true",
```

Comment:
Just for readability, you might consider adding more underscores.

Reply:
Will fix in my next PR, don't feel like babysitting CI for a couple of hours for this ...